odoo/addons/base/models/res_currency.py

```diff
@@ -185,6 +185,12 @@ def _num2words(number, lang):
                         )
         return amount_words
 
+    def amount_to_string(self, amount):
```

Comment:
There are already two tools for that: `tools.format_amount` & `tools.formatLang`, a third one will only add more confusion

Reply:
IMO, all the tooling related to currencies should be on `Model<res.currency>`, this centralizes everything and removes the need to import methods and pass the environment every single time.
But ok, maybe not for this time.